src/client/wrappers/box/BoxWrapper.js

```diff
@@ -7,7 +7,7 @@ import { use3Box, useLinkedData } from '../../hooks'
 
 const BoxWrapper = ({ account, children, onSignatures, isViewMode }) => {
   const { boxes, dispatch } = use3Box(account, onSignatures)
-  useLinkedData(boxes)
+  useLinkedData()
```

Comment:
Confused at why we changed the signature of this?

Reply:
...and I just changed it again:

`const useLinkedData = (boxes, dispatch, ethereumAddress) => {`

boxes returned as parameter, and I added dispatch, so the method can inform about exceptions via state (as everything else does).